lib/internal/Magento/Framework/Search/Response/QueryResponse.php

```diff
@@ -65,4 +74,12 @@ public function getAggregations()
     {
         return $this->aggregations;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTotalCount()
```

Comment:
@buskamuza @melnikovi can you please take a look if we can allow adding this method to the framework. It appears there is no other way to return the total count of the search result.

Reply:
This class has count method already, that seem to have similar purpose. Are there cases when count that it uses is not efficient?

Reply:
@melnikovi Count method returns a total count of the search result items of the current page.
We make getTotalCount for the return of all pages items count.